handlers/customer/handler.go

```diff
@@ -94,14 +80,7 @@ func GetByID(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func UpdateByID(w http.ResponseWriter, r *http.Request) {
-	db, err := drivers.ConnectToSQL()
-	if err != nil {
-		w.WriteHeader(http.StatusInternalServerError)
-		return
-	}
-	defer db.Close()
-
+func (h handler) UpdateByID(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
 	id := params["id"]
```

Comment:
covert ID to integer before passing it around

Reply:
done